tests/features/site.feature

```diff
@@ -11,7 +12,8 @@ Feature: site
 
   Scenario: Site Connection Mode
     @vcr site-connection-mode
-    Given a site named "[[test_site_name]]"
+    Given I am authenticated
+    And a site named "[[test_site_name]]"
     When I run "terminus site connection-mode --site=behat-test --env=dev"
```

Comment:
Need to use
` When I run "terminus site connection-mode --site=behat-test --env=dev --set=git"`
`And I run "terminus site connection-mode --site=behat-test --env=dev"`